lib/Dependency.js

```diff
@@ -48,4 +48,17 @@ class Dependency {
 }
 Dependency.compare = (a, b) => compareLocations(a.loc, b.loc);
 
+/** @typedef {Object} Position
+ *  @property {number} column
+ *  @property {number} line
+ */
+
+/** @typedef {Object} Loc
+ *  @property {Position} start
+ *  @property {Position} end
+ */
+
+exports Position;
```

Comment:
you don't need to export types

Reply:
I'll remove the exports.. and leave everything as is.

Reply:
@sokra done.